python/llm/example/GPU/PyTorch-Models/Model/llama2/low_memory_generate.py

```diff
@@ -0,0 +1,354 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
add license of original code and adaptation source in this file.

Reply:
The original code is also licensed under [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0) . Added adaptation source in the file.